src/components/digitalSpecimen/tourSteps/DigitalSpecimenTourSteps.tsx

```diff
@@ -0,0 +1,102 @@
+/* Import Dependencies */
+import { Steps } from 'intro.js-react';
+
+/* Import Config */
+import StepsConfig from 'app/config/StepsConfig';
+
+/* Import Hooks */
+import { useAppSelector, useAppDispatch } from 'app/Hooks';
+
+/* Import Store */
+import { getTourTopic, setTourTopic } from 'redux-store/GlobalSlice';
+
+/* Import Sources */
+import DigitalSpecimenTourStepsText from 'sources/tourText/digitalSpecimen.json';
+
+
+/* Props Type */
+type Props = {
+    SetSelectedTabIndex: Function
+};
+
+
+/**
+ * Component that renders the tour steps for the digital specimen page
+ * @param SetSelectedTabIndex Function to set the selected tab index
+ * @returns JSX Component
+ */
+const DigitalSpecimenTourSteps = (props: Props) => {
+    const { SetSelectedTabIndex } = props;
+
+    /* Hooks */
+    const dispatch = useAppDispatch();
+
+    /* Base variables */
+    const tourTopic = useAppSelector(getTourTopic);
+    const digitalSpecimenSteps = DigitalSpecimenTourStepsText.digitalSpecimen;
+    const { options } = StepsConfig();
+    let steps: {
+        intro: string,
+        element?: string
+    }[] = [];
+
+    /* Construct Intro.js steps for digital specimen page */
+    digitalSpecimenSteps.forEach((step, index) => {
+        steps.push({
+            intro: step,
+            element: `.tourDigitalSpecimen${index + 1}`
+        });
+    });
+
+    return (
+        <Steps enabled={tourTopic === 'digitalSpecimen'}
+            steps={steps}
+            initialStep={0}
+            onBeforeChange={(nextIndex) => {
```

Comment:
bit repetitive here with setting the tabs from the indexes, you could do something a bit more concise: 

```
const tabIndexMap = {
    6: 0, // Step 7: set tab to digital specimen
    7: 1, // Step 8: set tab to digital media
    8: 2, // Step 9: set tab to events
    9: 3, // Step 10: set tab to identifications
    10: 4, // Step 11: set tab to entity relationships
    11: 5  // Step 12: set tab to assertions
};

const handleStepChange = (nextIndex) => {
    if (tabIndexMap[nextIndex] !== undefined) {
        SetSelectedTabIndex(tabIndexMap[nextIndex]);
    }
};

```

Reply:
Fixed